Directory.Packages.props

```
@@ -93,33 +93,33 @@
     <PackageVersion Include="Microsoft.Extensions.Http.Resilience" Version="$(MicrosoftExtensionsHttpResiliencePackageVersion)" />
     <PackageVersion Include="Microsoft.Extensions.TimeProvider.Testing" Version="$(MicrosoftExtensionsTimeProviderTestingVersion)" />
     <!-- external dependencies -->
-    <PackageVersion Include="Confluent.Kafka" Version="2.5.0" />
+    <PackageVersion Include="Confluent.Kafka" Version="2.5.2" />
     <PackageVersion Include="Dapper" Version="2.1.44" />
     <PackageVersion Include="DnsClient" Version="1.8.0" />
-    <PackageVersion Include="Elastic.Clients.Elasticsearch" Version="8.14.7" />
-    <PackageVersion Include="Google.Protobuf" Version="3.27.2" />
+    <PackageVersion Include="Elastic.Clients.Elasticsearch" Version="8.15.2" />
+    <PackageVersion Include="Google.Protobuf" Version="3.27.3" />
     <PackageVersion Include="Grpc.AspNetCore" Version="2.65.0" />
     <PackageVersion Include="Grpc.Net.ClientFactory" Version="2.65.0" />
     <PackageVersion Include="Grpc.Tools" Version="2.65.0" />
     <PackageVersion Include="Humanizer.Core" Version="2.14.1" />
-    <PackageVersion Include="KubernetesClient" Version="14.0.2" />
+    <PackageVersion Include="KubernetesClient" Version="14.0.9" />
     <PackageVersion Include="Microsoft.Data.SqlClient" Version="5.2.1" />
     <PackageVersion Include="Microsoft.FluentUI.AspNetCore.Components" Version="4.9.3" />
     <PackageVersion Include="Microsoft.FluentUI.AspNetCore.Components.Icons" Version="4.9.3" />
     <PackageVersion Include="Milvus.Client" Version="2.3.0-preview.1"/>
-    <PackageVersion Include="MongoDB.Driver" Version="2.27.0" />
-    <PackageVersion Include="MongoDB.Driver.Core.Extensions.DiagnosticSources" Version="1.4.0" />
+    <PackageVersion Include="MongoDB.Driver" Version="2.28.0" />
+    <PackageVersion Include="MongoDB.Driver.Core.Extensions.DiagnosticSources" Version="1.5.0" />
```

```suggestion
    <PackageVersion Include="MongoDB.Driver" Version="2.27.0" />
    <PackageVersion Include="MongoDB.Driver.Core.Extensions.DiagnosticSources" Version="1.4.0" />
```

This update can't happen yet.